apitools/base/py/credentials_lib.py

```diff
@@ -352,7 +352,7 @@ def _ScopesFromMetadataServer(self, scopes):
     def GetServiceAccount(self, account):
         relative_url = 'instance/service-accounts'
         response = _GceMetadataRequest(relative_url)
-        response_lines = [line.rstrip('/\n\r')
+        response_lines = [line.rstrip(b'/\n\r').decode('utf-8')
```

Comment:
Looks like #271 fixes this Py3 incompatibility, but not the one in _do_refresh_request below.